server/src/app.js

```diff
@@ -23,8 +25,15 @@ app.use(cors(corsOptions));
 
 // Cookies
 import cookieParser from 'cookie-parser';
+import { deleteUserIfExpired } from './models/User.js';
 app.use(cookieParser());
 
+// We use cron to schedule tasks
+cron.schedule('00 * * * *', () => {
+  // running a task every hour
```

Comment:
How can the regularity of the scan be reconfigured? Say, reduce it from an hour to a day.

Reply:
Sorry, I forgot to change it, I am going to change it